src/clj/ataru/applications/excel_export.clj

```diff
@@ -106,22 +106,26 @@
           reviews)))))
 
 (defn- application-review-notes-formatter
-  [review-notes]
-  (->> review-notes
-       (map (fn [{:keys [created-time notes hakukohde first-name last-name]}]
-              (str
-                (time-formatter created-time)
-                " "
-                (when (and first-name last-name)
-                  (str
-                    first-name
-                    " "
-                    last-name))
-                (when hakukohde
-                  (str " (hakukohde " hakukohde ")"))
-                ": "
-                notes)))
-       (clojure.string/join ",\n")))
+  ([review-notes]
+    (application-review-notes-formatter nil review-notes))
+  ([state-to-select review-notes]
+   (->> (if state-to-select
+          (filter #(= state-to-select (:state-name %)) review-notes)
+          (remove #(= "eligibility-state" (:state-name %)) review-notes))
```

Comment:
Voisko tää olla `(remove #(some? (:state-name %)) review-notes)`? Pitäis teoriassa toimia myös jos `:state-name` on jotain muuta kuin `"eligibility-state"`.